test/compress/issue-203.js

```diff
@@ -26,9 +26,7 @@ compress_new_function_with_destruct: {
     }
     expect: {
         Function("a", "[b]", "return a");
-        Function("a", "{bb}", "return a");
-        Function("[[a]]", "[{bb}]", 'return a');
+        Function("a", "{bb:b}", "return a");
+        Function("[[a]]", "[{bb:b}]", 'return a');
```

Comment:
Reading https://github.com/mishoo/UglifyJS2/issues/203#issuecomment-17861551 - I guess if it was reluctantly implemented, it's probably harmless to hide it behind its own flag instead of `unsafe`.

Although can we get a may be more descriptive, but definitely shorter name than `unsafe_Function`, please? :sweat_smile: